packages/codegen/src/config.test.ts

```diff
@@ -0,0 +1,47 @@
+import { Config, defaultConfig, FieldConfig, isFieldIgnored } from "./config";
+import { makeEntity } from "./EntityDbMetadata";
+
+describe("config", () => {
+  describe("isFieldIgnored", () => {
+    // A little hack to test the case of `ignore: false`, since it is actually defined as `ignore?: true`
+    const falseAsTrue = (false as boolean) as true;
+
+    it.each([
+      ["fields key is omitted all together", newAuthorConfig()],
```

Comment:
Just from reading this, I was unclear what the "then" condition was going to be (it's `false`), i.e. the test name is "config isFieldIgnored fields key is omitted all together" ...

Like are we ignoring here...

Maybe something more like

* "does not ignore when no config is provided"
* "does not ignore when config is set but ignore is unset"
* "does not ignore when ignore is false"
* "does ignore when ignore is true"

I.e. I might work in the next `it` into this each, just b/c it feels like an arbitrary break up for "here are all of the `false` cases" ... when I readed this, where is the true case?... oh here it is, it's the next `it` test. Seems like `it.each` should try to cover all of the boundary cases. 

Granted, this is a trade-off based on the complexity of the mini-DSL/tuples we end up needing to build for `it.each` to accept. And, to be fair, we do having an outer `describe(isFieldIgnored)` to encompass them all. So that's fine, just talking out loud.

Reply:
...also I'll re-iterate my usual `it.each` bias that I'm not sure this is really a net improvement in readabiliity and long-term dev UX vs. each `it` blocks.

Reply:
Cleaned up the messages. TY 👍